src/main/java/application/App.java

```diff
@@ -0,0 +1,42 @@
+package application;
+import http_server.RequestParser;
+import http_server.Response;
+import http_server.Server;
+import http_server.Router;
+
+public class App {
+    static Router app;
+    static Server server;
+
+    public static void main(String args[]) {
+        app = new Router();
+        createRoutes();
+
+        server = new Server(setPortNumber(args), app);
+        server.start();
+    }
+
+    private static void createRoutes() {
+        app.get("/simple_get", (RequestParser req, Response res) -> {});
```

Comment:
I super dig this structure.  Did you base it on something?  (My gut check is mapping this to `express` right now but maybe there's something else like this.)

Reply:
I based the route creation on Express, but for the overall structure of App class, it came about naturally to get the whole thing working. But I definitely noticed large similarities to Express and see why they structure it the same way!